velox/experimental/cudf/exec/CudfHashJoin.cpp

```diff
@@ -367,6 +374,9 @@ RowVectorPtr CudfHashJoinProbe::getOutput() {
   }
   VELOX_NVTX_OPERATOR_FUNC_RANGE();
 
+  if (finished_) {
```

Comment:
Should we combine with the below, to `if(finished_ || !input || !hashObject_.has_value())`?